web-dapp/server-lib/logger.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
     },
     error: (msg) => {
         let logDate = new Date().toISOString();
+        logger.info(logDate, msg);
```

Comment:
Not sure about this. Logging the same message with two different levels doesn't feel right.

If you want to have all the content in a file, you can do something like:

`npm start 2>&1 | tee log.txt`

Reply:
We don't have `npm start` on production server, but I've found how to merge logs there too